packages/neuron-wallet/src/block-sync-renderer/tx-status-listener.ts

```diff
@@ -74,10 +75,7 @@ const trackingStatus = async () => {
 }
 
 export const register = () => {
-  // every 5 seconds
-  setInterval(async () => {
-    // Disable debug output as it's too annoying.
-    // logger.debug("periodically status tracking ...")
+  interval(5000).pipe().subscribe(async () => {
```

Comment:
What does the empty `pipe` do?

Reply:
It is actually unnecessary. The empty pipe is now removed.